en/docs/reference/connectors/mongodb-connector/mongodb-connector-example.md

```diff
@@ -30,43 +30,39 @@ If you want to connect to MongoDB Atlas, follow the steps mentioned below to get
 
 4. Select Java from the **Driver** menu.
 
-5. Select the correct driver version from the **Version** menu.
+5. Select the correct driver version from the **Version**(`3.3`) menu.
```

Comment:
Why are we mentioning version here? If a particular version is not supported add a note.